sigs/storage-charter.md

```diff
@@ -0,0 +1,125 @@
+# CNCF Storage SIG Charter
+
+Primary Author: Quinton Hoole
+
+2019-04-23
+
+Reviewed and contributed to by Saad Ali, Alex Chircop,
+Xing Yang, Xiang Li, and other members of the CNCF Storage Working
+Group and community.
+
+# Introduction
+
+This is the charter referred to in [CNCF
+SIGs](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#sig-charter)
+by the CNCF TOC.
+
+# Areas Considered In Scope
+
+Storage systems and approaches suitable for and commonly used in
```

Comment:
I feel like there is a verb or two missing here - doing what with storage systems and approaches? Given the next paragraph maybe it is as simple as "understanding", but maybe also "documenting"?

Reply:
Agreed, will add a verb.

Reply:
Sorry to be picky @quinton-hoole I would feel more comfortable with casting my vote if we had something to resolve this comment (especially as it could be used as a model charter for other SIGs).

Reply:
@lizrice No problem.  I've added a verb to make it a proper sentence.  The verbless phrase also grated my grammar education sensibilities :-)

To avoid duplicating what's already in the next paragraph, and the mission statement section, I just added "is considered in scope".